grpc_message_getter.js

```diff
@@ -0,0 +1,293 @@
+const {ClientError} = require('./client_error');
+const {EllipticSigner, WebCryptoSigner} = require('./signer');
+const {TextEncoder} = require('./request/builder');
+const {StatusCode} = require('./status_code');
+
+class GrpcMessageGetter {
```

Comment:
I feel it has nothing  to do with `grpc` but does it ?

Reply:
We are building the buffer of the grpc request isn't it? or am i wrong...Or maybe we can change it to be `SignedMessageGetter`

Reply:
I think it just serializes entries and it has nothing to do with grpc, does it ?

Reply:
Right, after a discussion with Plenty , yes not really has any relation to grpc, it is more like a serializer.

Another thing is that after we realized that grpc natively has a method that return us the binary array of the request by `request.serializeBinary()`, [here is the short documentation about it](https://github.com/scalar-labs/scalardl-node-client-sdk/blob/master/scalar_pb.js#L145-L153), we might not even the `GrpcMessageGetter` class, we can just call the method `getByteArray()` to get the binary array...thought?

Reply:
Oh, I think that might be better !
then creating `executeContract(byte[] request)` in the client SDK might be good enough.
Let's use `serializeBinary` !